entity-framework/core/cli/dotnet.md

```diff
@@ -262,7 +262,9 @@ dotnet ef dbcontext scaffold Name=ConnectionStrings:Blogging Microsoft.EntityFra
 The following example skips scaffolding an `OnConfiguring` method. This can be useful when you want to configure the DbContext outside of the class. For example, ASP.NET Core apps typically configure it in Startup.ConfigureServices.
 
 ```dotnetcli
-dotnet ef dbcontext scaffold "Server=(localdb)\mssqllocaldb;Database=Blogging;User Id=myUsername;Password=myPassword;" Microsoft.EntityFrameworkCore.SqlServer --no-onconfiguring
+dotnet ef dbcontext scaffold "Server=(localdb)\mssqllocaldb;Database=Blogging;User Id=myUsername;Password=\0;" Microsoft.EntityFrameworkCore.SqlServer --no-onconfiguring
+
+In the preceding code, replace `\0` with a valid strong password.
```

Comment:
That's probably still a valid password. How about I use `;|;` or `;A;` and tell them to replace `;A` with a ...

Reply:
Why not simply `***`?

Reply:
> Why not simply `***`?

Because that's a valid PW and can be copy/pasted. And customer copy/paste.